resources/views/pages/forums/post/[ForumTopicComment]/edit.blade.php

```diff
@@ -45,16 +37,16 @@
     <x-section>
         <x-base.form action="{{ url('request/forum-topic-comment/update.php') }}">
             <div class="flex flex-col gap-y-3">
-                <x-base.form.input type="hidden" name="comment" value="{{ $foundPost->ID }}" />
+                <x-base.form.input type="hidden" name="comment" value="{{ $forumTopicComment->id }}" />
 
-                <x-base.form.input label="{{ __res('forum', 1) }}" readonly value="{{ $topic->forum->title }}" inline :fullWidth="false" />
-                <x-base.form.input label="{{ __res('author', 1) }}" readonly value="{{ $topic->user?->User ?? 'Deleted user' }}" inline :fullWidth="false" />
-                <x-base.form.input label="{{ __res('forum-topic', 1) }}" readonly :value="$topic->title" inline />
+                <x-base.form.input label="{{ __res('forum', 1) }}" readonly value="{{ $forum->title }}" inline :fullWidth="false" />
+                <x-base.form.input label="{{ __res('author', 1) }}" readonly value="{{ $forumTopic->user?->display_name ?? 'Deleted user' }}" inline :fullWidth="false" />
+                <x-base.form.input label="{{ __res('forum-topic', 1) }}" readonly :value="$forumTopic->title" inline />
 
                 <x-base.form.textarea
                     id="input_compose"
                     label="{{ __res('message', 1)}} "
-                    :value="$foundPost->Payload"
+                    :value="$forumTopicComment->body"
```

Comment:
How does `forumTopicComment` get passed here? Doesn't it have to be in the `$view->with([` list?

Reply:
Folio treats all dynamic fields (`[thing]`) as globally-available context values. They can be "peeked" into in `render()`. More docs here: https://laravel.com/docs/11.x/folio#route-model-binding

```php
// [ForumTopicComment] --> $forumTopicComment
render(function (View $view, ForumTopicComment $forumTopicComment) { ... });
```

However, there's no harm in also putting it in `with()`. I also re-added `@props()` for improved documentation.